### PR TITLE
Custom resolution option

### PR DESCRIPTION
**Pull request by sliverc**
Added a configuration so it is possible to configure a resolution on thin client. This can be useful on thin clients where automatic resolution is not set correctly.

When a resolution is configured for a thin client it will use xrandr to change it before opening remote viewer.

Some real life testing will need to be done before this can be merged. On existing database there is a script database-layout-update-001.sql to update it.

@winged 
Do not merge this just yet but can you have a look and say what you think?

### CODE REVIEW

File: sample_config/database-layout-update-001.sql
Comment (winged) [inline sample_config/database-layout-update-001.sql:71]: Can you clean up the indentation and code layout a bit, and maybe add a few comments? See our SQL style guides for examples. This would help understanding this long VIEW

File: sample_config/database-layout-update-001.sql
Comment (winged) [inline sample_config/database-layout-update-001.sql]: One could argue getting rid of all the permission stuff altogether in the DB layout. This would give users more freedom in building the DB, and won't break if we someday choose to use another DB username.

File: sample_config/database-layout.sql
Comment (winged) [inline sample_config/database-layout.sql]: Why the indentation?

File: sample_config/database-layout.sql
Comment (winged) [inline sample_config/database-layout.sql]: Much better than the oneliner from before, but again - use our SQL style guide :)

File: virtesk-tc-connectspice/connect_spice_client.py
Comment (winged) [inline virtesk-tc-connectspice/connect_spice_client.py:95]: Future Cleanup idea: Use a row factory that supports dict-like access..

**Review by winged — CHANGES_REQUESTED** [review]
Some minor changes and questions, but overall looks good

**Comment by sliverc** [comment]
@winged 
Adjusted SQL script and added some comments and an additional TODO.
Say whether it looks good or not but do not merge yet... I will send then an email to Dirk so he can test the changes on the prod system. Once successful we can merge.

**Pull request merged**
